searx/webutils.py

```diff
@@ -134,3 +136,30 @@ def is_flask_run_cmdline():
     if len(frames) < 2:
         return False
     return frames[-2].filename.endswith('flask/cli.py')
+
+
+DEFAULT_GROUP_NAME = 'others'
```

Comment:
Single-point of definition: should we better use the value from `searx.engines.OTHER_CATEGORY`? --> In webapp.py there is a `from searx.engines import  OTHER_CATEGORY, DEFAULT_GROUP_NAME`.

Do we need two values, one for default one for the "OTHER" tab in the preferences?

Reply:
I think for the tab "Other" fits better because general, news, map, music, it, science, social media are all singular.
Whereas the subcategories are almost all plural, e.g. packages, repos & others, apps & others, dictionaries & others.

Reply:
But we do not need a "DEFAULT_GROUP_NAME" I think.

PLEASE: do not close my remarks as "resolved conversation" .. I will do it .. and I think @dalf, @mrpaulblack should have a chance to read my remarks ...

Reply:
>But we do not need a "DEFAULT_GROUP_NAME" twice I think.

OTHER_CATEGORY and DEFAULT_GROUP_NAME fulfill different functions. Using a single variable for different things seems like bad style to me. (Besides that if we want different UI text like I explained we need different variables).

>PLEASE: do not close my remarks as "resolved conversation"

Right, sorry about that. I think it's a bit annoying how GitHub intermixes code review with PR discussion, it makes the comments on the PR hard to follow.

Reply:
>  it makes the comments on the PR hard to follow.

Yes, its hard to follow .. I understand.

>> But we do not need a "DEFAULT_GROUP_NAME" twice I think.
> OTHER_CATEGORY and DEFAULT_GROUP_NAME fulfill different functions. 

Ooops sorry, my fail: I thought you have defined DEFAULT_GROUP_NAME twice .. but thats not the case.